src/main/java/org/zalando/nakadi/security/Client.java

```diff
@@ -1,22 +1,29 @@
 package org.zalando.nakadi.security;
 
-public interface Client {
+import org.zalando.nakadi.exceptions.IllegalScopeException;
 
-    Client PERMIT_ALL = clientId -> true;
+import java.util.Set;
 
-    boolean is(String clientId);
+public class Client implements IClient {
```

Comment:
Let's decide https://docs.google.com/spreadsheets/d/1p2GTg-fgrNGQBkhdM1yzKwvWOXvFL4FyiOoGtQJqTxI/edit#gid=0
BTW why have you renamed it?